themes/classic/templates/checkout/_partials/cart-detailed-actions.tpl

```diff
@@ -0,0 +1,15 @@
+<div class="checkout cart-detailed-actions card-block">
+  {if $cart.minimalPurchaseRequired}
+    <div class="alert alert-warning" role="alert">
+      {$cart.minimalPurchaseRequired}
+    </div>
+    <div class="text-xs-center">
+      <button type="button" class="btn btn-primary disabled" disabled>{l s='Checkout'}</button>
```

Comment:
That's why I don't like bootstrap, what is the point of having `.disabled` and attr disabled :D